draft-ietf-tls-tls13.md

```diff
@@ -2382,12 +2382,13 @@ The "extension_data" field of this extension contains an
 
 %%% Key Exchange Messages
 
+       struct {} Empty;
```

Comment:
I doubly checked with my validator.

- `Empty`s are used only here.
- The syntax of this branch is valid.